### PR TITLE
ecs_task_definitions_implementation

### PR DESCRIPTION
**Pull request by amanenk**

### CODE REVIEW

File: resources/ecs_task_definitions.go [hunk 1]
```diff
@@ -0,0 +1,914 @@
...
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:     "tags",
```
Comment (roneli) [inline resources/ecs_task_definitions.go:37]: missing description

File: resources/ecs_task_definitions.go [hunk 2]
```diff
@@ -0,0 +1,914 @@
...
+				Type:     schema.TypeJSON,
+				Resolver: resolveEcsTask_definitionTags,
```
Comment (roneli) [inline resources/ecs_task_definitions.go:39]: nit: resolver name is not golang convention should be camelcase

File: resources/ecs_task_definitions.go [hunk 3]
```diff
@@ -0,0 +1,914 @@
...
+					},
+					{
+						Name:        "working_directory",
+						Description: "The working directory in which to run commands inside the container",
+						Type:        schema.TypeString,
+					},
+				},
+			},
+			{
+				Name:        "aws_ecs_task_definition_requires_attributes",
```
Comment (roneli) [inline resources/ecs_task_definitions.go:463]: nit: maybe make this a json as well?

**Review by roneli — CHANGES_REQUESTED** [review]
Looks good, suggested some changes.

File: resources/ecs_task_definitions.go (hunk 2 above)
Comment (amanenk) [reply resources/ecs_task_definitions.go:39]: this fix is in one of my PRs to cq-gen :)

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**